engine/DJI_E305.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!--bldc_engine name="DJI E305"-->
+  <brushless_dc_motor name="DJI E305">
+  <!-- E305 Motor is a DJI 2312E - 960 -->
+  <power unit="WATTS"> 320 </power>
+  <velocityconstant> 960 </velocityconstant>
+  <torqueconstant> 6.1217 </torqueconstant>
```

Comment:
Bit confused about `<torqueconstant>`. Is this supposed to be a user supplied value for `Kq` as per - http://web.mit.edu/drela/Public/web/qprop/motor1_theory.pdf

Although this element isn't read in by `FGBrushLessDCMotor::FGBrushLessDCMotor()`, instead we have the following in the header file.

```c++
constexpr double TorqueConstant = 60 / (2 * M_PI * NMtoftpound);
```

So what is the relationship between them?

Reply:
According to theory Kq (torque constant) can be assumed to be 1/Kv (speed constant ) if we define speed  in rad/sec torque in Newton*Meter. 
Therefore since torque is in ft*pound a unit conversion factor is necessary (the conversion factor is `TorqueConstant`)

Reply:
The naming is confusing, nomenclature from the motor theory document:

![image](https://user-images.githubusercontent.com/824856/150688229-e26d35ea-bdb1-4bed-91d2-257b42bd669d.png)

However your variable, `TorqueConstant` is a constant conversion factor. Whereas when I was initially reading the source code and comparing it to the motor model document I assumed your `TorqueConstant` was referring to Kq.

![image](https://user-images.githubusercontent.com/824856/150688367-f9aab696-a286-4b20-9575-1a0a0d52c35b.png)

Also given this:

![image](https://user-images.githubusercontent.com/824856/150688516-ad7aec2d-9d5d-4b15-93b3-b168c4b57ba2.png)

Shouldn't we allow the user to optionally specify the torque constant Kq if they want to model the efficiency more accurately?

Reply:
Yes i agree....